### DISCUSSION TITLE
Finally moved the ECS to this new repository!

### DISCUSSION THREAD

**Discussion by friflo** [friflo/Friflo.Engine.ECS#1] [Announcements]
[Friflo.Engine.ECS](https://github.com/friflo/Friflo.Json.Fliox/blob/main/Engine/README.md) was devloped within the `Engine` sub folder of a repository containing an ORM.

To avoid ECS/ORM confusion the ECS is now maintained here in its own repository.

Discussions in the old repo can be found [here](https://github.com/friflo/Friflo.Json.Fliox/discussions)
Issues in the old repo [here](https://github.com/friflo/Friflo.Json.Fliox/issues)

Motivation for new repo:
Moving the ECS to its own repo was #1 feedback.
In multiple reddit comments and in discussion [Ecs separation](https://github.com/friflo/Friflo.Json.Fliox/discussions/36)

**Comment by yangtzehina** [comment]
Good Job!!

**Reply by friflo** [reply]
Moving a repo is pain - so many links to update.
